### ISSUE TITLE
Enhancement suggestion: Add "Create Wallet" on section 3.3 since a default wallet is no longer automatically created by Bitcoin Core.

### ISSUE THREAD

**Issue by felipegontijo** [BlockchainCommons/Learning-Bitcoin-from-the-Command-Line#202]
Found out about this when running the `bitcoin-cli getnewaddress` command:

```
error code: -18
error message:
No wallet is loaded. Load a wallet using loadwallet or create a new one with createwallet. (Note: A default wallet is no longer automatically created)
```

I therefore believe a "Create Wallet" section could be added before "Create an Address," to include such explanation and the `bitcoin-cli createwallet` and `bitcoin-cli loadwallet` commands.

**Comment by Pdptk** [comment]
You are running code from the console in a virtual machine?

El dt., 13 abr. 2021, 13.39, Felipe Gontijo ***@***.***> va
escriure:

**Comment by felipegontijo** [comment]
I am running GordianServer-macOS and interacting with it via terminal. Doing everything in testnet by the way.

**Comment by Pdptk** [comment]
MacOs distros are always tiding bugs with bitcoin claims. Don't you know?

El dt., 13 abr. 2021, 13.47, Felipe Gontijo ***@***.***> va
escriure:

**Comment by felipegontijo** [comment]
I'm afraid I don't understand

**Comment by Pdptk** [comment]
Sorry this message wasn't for you. Excuse me.

El dc., 21 abr. 2021, 13.20, Felipe Gontijo ***@***.***> va
escriure:

**Comment by shannona** [comment]
The tutorial is intended for a Debian install, and although we certainly hope it works elsewhere, especially on something like Gordian Server, it may not.

I'm flagging this to see if a change has also broken the Debian install, or if it's a MacOS-only issue.

**Comment by liorko87** [comment]
The change is in bitcoin 0.21.0 and not related to particular OS.
Since 0.21.0 you must load the wallet by calling the command `loadwallet` at the beginning or by adding this line to your botcoin config file:
```
wallet=<wallet_name>
```

**Comment by shannona** [comment]
Thanks for the clarification, @liorko87, flagging this as a TODO for 2.1 or 3.0.

**Comment by shannona** [comment]
Resolved by @namcios in #479.
